app/javascript/components/App.js

```diff
@@ -23,7 +22,6 @@ function App() {
             <Route exact path="/" render={() => <Main child={<LandingPage />} />} />
```

Comment:
Another way to write this is:

```js
<Main>
  <LandingPage />
</Main>
```

And from inside the `Main` component, you can access `<LandingPage />` in the `children` prop.